kratos/tests/cpp_tests/utilities/test_qef_utility.cpp

```diff
@@ -72,95 +72,80 @@ namespace {
         return p_triangle->pGetGeometry();
-    } 
+    }
 } //unnamed namespace
 
-KRATOS_TEST_CASE_IN_SUITE(CalculateNormal, KratosCoreFastSuite) 
+KRATOS_TEST_CASE_IN_SUITE(QuadraticErrorFunction0dof, KratosCoreFastSuite)
 {
-    // Generate a triangles
-    std::vector<std::vector<double>> triangle1{{0,0,0},{1,0,0},{0,1,0}};    
-    GeometryPtrType p_triangle = QuadraticErrorFunctionGenerateTriangle3D3(triangle1);
-
-    // Call the normal utility
-    const array_1d<double,3> normal = QuadraticErrorFunction::CalculateNormal(*p_triangle);
-
-    //Expected output of the function
-    KRATOS_EXPECT_EQ(normal[0], 0.0);
-    KRATOS_EXPECT_EQ(normal[1], 0.0);
-    KRATOS_EXPECT_EQ(normal[2], 1.0);
-}
-
-KRATOS_TEST_CASE_IN_SUITE(QuadraticErrorFunction0dof, KratosCoreFastSuite) 
-{
-    // A voxel crossed by a straight plane with only 1 nodes inside the volume 
-    std::vector<double> distances{1, -1, -1, -1, -1, -1, -1, -1};   
+    // A voxel crossed by a straight plane with only 1 nodes inside the volume
+    std::vector<double> distances{1, -1, -1, -1, -1, -1, -1, -1};
     GeometryPtrType p_voxel = QuadraticErrorFunctionGenerateHexahedra3D8(distances);
 
     // Generate the intersecting triangles
-    std::vector<std::vector<double>> triangle1{{-1,0.5,-0.95},{-0.95,0.5,-1.05},{-1.05,0.5,-1.05}}; 
-    std::vector<std::vector<double>> triangle2{{-1,-0.95,0},{-0.95,-1.05,0},{-1.05,-1.05,0}}; 
-    std::vector<std::vector<double>> triangle3{{0.75,-0.95,-1},{0.75,-1.05,-0.95},{0.75,-1.05,-1.05}}; 
+    std::vector<std::vector<double>> triangle1{{-1,0.5,-0.95},{-0.95,0.5,-1.05},{-1.05,0.5,-1.05}};
+    std::vector<std::vector<double>> triangle2{{-1,-0.95,0},{-0.95,-1.05,0},{-1.05,-1.05,0}};
+    std::vector<std::vector<double>> triangle3{{0.75,-0.95,-1},{0.75,-1.05,-0.95},{0.75,-1.05,-1.05}};
 
     GeometryPtrType p_triangle1 = QuadraticErrorFunctionGenerateTriangle3D3(triangle1);
     GeometryPtrType p_triangle2 = QuadraticErrorFunctionGenerateTriangle3D3(triangle2);
     GeometryPtrType p_triangle3 = QuadraticErrorFunctionGenerateTriangle3D3(triangle3);
 
     GeometryArrayType array1;
-    array1.push_back(p_triangle1); 
+    array1.push_back(p_triangle1);
     array1.push_back(p_triangle2);
     array1.push_back(p_triangle3);
 
     const array_1d<double,3> point = QuadraticErrorFunction::QuadraticErrorFunctionPoint(*p_voxel,array1);
 
     KRATOS_EXPECT_NEAR(point[0], 0.75, 1e-8);
     KRATOS_EXPECT_NEAR(point[1], 0.5, 1e-8);
-    KRATOS_EXPECT_NEAR(point[2], 0.0, 1e-8);        
+    KRATOS_EXPECT_NEAR(point[2], 0.0, 1e-8);
 }
 
-KRATOS_TEST_CASE_IN_SUITE(QuadraticErrorFunction1dof, KratosCoreFastSuite) 
+KRATOS_TEST_CASE_IN_SUITE(QuadraticErrorFunction1dof, KratosCoreFastSuite)
 {
     //A voxel crossed by a straight plane with only 2 nodes inside the volume
-    std::vector<double> distances{1, 1, -1, -1, -1, -1, -1, -1};   
+    std::vector<double> distances{1, 1, -1, -1, -1, -1, -1, -1};
```

Comment:
I think distances are not necessary for these tests and can be also cleaned up. (I do not remember why I added them in the first place). In case they are indeed necessary, I would suggest to follow the standard sign convention, negative inside the volume and positive outside.

Reply:
No, I mean adding the distances to the voxel. We do the whole AddNodalSolutionStepVariable and initialization, but then this distance values are not used in the QEF utility, if I am not wrong.